### PR TITLE
Feature1 - Add Vegetable 

### PR DESCRIPTION
**Pull request by jegan45**

### CODE REVIEW

File: src/main/java/in/jegan/service/VegetableManager.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import in.jegan.validater.VegetableManagerValidater;
+
+
+public class VegetableManager {
+	private VegetableManager()
+	{
+		//defaultconstructor;
+	}
+
+	private  static List<Vegetable> vegetableList = new ArrayList<>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/jegan/service/VegetableManager.java]: Add final

File: src/main/java/in/jegan/service/VegetableManager.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	public static boolean addVegetables(Vegetable vegetable)
+	{
+		boolean added = false;
+		try {
+			if(VegetableManagerValidater.checkForNullandEmpty(vegetable) && VegetableManagerValidater.checkNotNumeric(vegetable)  && VegetableManagerValidater.checkForPriceInvalidNumandNull(vegetable) && VegetableManagerValidater.checkForQuantityInvalidNumandNull(vegetable))
+			{
+				vegetableList.add(vegetable);
+				 added=true;
+			}
+		} catch (Exception e) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/jegan/service/VegetableManager.java:46]: Add e.printStackTrace

File: src/main/java/in/jegan/service/VegetableManager.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	/**
+	 * This method is used to add vegetables and returns true only the given conditions are true otherwise returns false
+	 * @param vegetable
+	 * @return
+	 */
+	public static boolean addVegetables(Vegetable vegetable)
+	{
+		boolean added = false;
+		try {
+			if(VegetableManagerValidater.checkForNullandEmpty(vegetable) && VegetableManagerValidater.checkNotNumeric(vegetable)  && VegetableManagerValidater.checkForPriceInvalidNumandNull(vegetable) && VegetableManagerValidater.checkForQuantityInvalidNumandNull(vegetable))
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/jegan/service/VegetableManager.java]: Check spelling Validator

File: src/test/java/in/jegan/Tester/AddVegetableManagerTest.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
+package in.jegan.Tester;
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/jegan/Tester/AddVegetableManagerTest.java]: Package name - follow naming convention

File: src/test/java/in/jegan/Tester/AddVegetableManagerTest.java [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+		Vegetable vegetable = new Vegetable("Tomato", -1, 25);
+		boolean added= VegetableManager.addVegetables(vegetable);
+		assertFalse(added);
+	}
+	
+	/**
+	 * This method is used to test invalid quantity value
+	 */
+	@Test
+	public void testForInvalidquantityInput()
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/jegan/Tester/AddVegetableManagerTest.java]: follow Naming convention

**Review by ghost — CHANGES_REQUESTED** [review]
Added Review comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**